apollo-router/src/state_machine.rs

```diff
@@ -262,9 +264,22 @@ where
                     Errored(FederatedServerError::ServiceCreationError(err))
                 })?;
 
+            let plugin_handlers: HashMap<String, Handler> = self
+                .router_factory
+                .plugins()
+                .iter()
+                .filter_map(|(plugin_name, plugin)| {
+                    plugin_name
+                        .starts_with("apollo.")
+                        .then(|| plugin.custom_endpoint())
```

Comment:
This check means "experimental" plugins won't be able to have custom_endpoints. Should they?

Reply:
fixed